sys/include/net/nanocoap.h

```diff
@@ -490,6 +538,23 @@ ssize_t coap_opt_add_uint(coap_pkt_t *pkt, uint16_t optnum, uint32_t value);
  */
 ssize_t coap_opt_finish(coap_pkt_t *pkt, uint16_t flags);
 
+/**
+ * @brief   Insert block2 option into buffer
+ *
+ * When calling this function to initialize a packet with a block2 option, the
+ * more flag must be set to prevent the creation of an option with a length too
+ * small to contain the size bit.
+ *
+ * @param[out]  buf         buffer to write to
+ * @param[in]   lastonum    number of previous option (for delta calculation),
+ *                          must be < 27
```

Comment:
Typo, should be 23, as in [e2d3564](https://github.com/bergzand/RIOT/pull/4/commits/e2d3564c1e0335fe6df8f37e2f20ee9830d772e6).

Reply:
oops :sob: